src/nvim/highlight_group.c

```diff
@@ -215,6 +215,7 @@ static const char *highlight_init_both[] = {
   "default link LspReferenceRead            LspReferenceText",
   "default link LspReferenceText            Visual",
   "default link LspReferenceWrite           LspReferenceText",
+  "default link LspReferenceTarget          LspReferenceText",
```

Comment:
Reviewing the LSP spec and search results, I'm wondering why we need a new highlight. It  seems like `DocumentHighlightKind.Text` perfectly fits the use-case here. @MariaSolOs @mfussenegger any preference?

Reply:
The only downside to this I can think of is that it would not be possible to disable highlights specifically for hover actions, since it is tied to hl group, but I don't know if anyone would even want to do this

Reply:
I prefer having the separate highlight group. I don't set `LspReferenceText` in my config because I find them to be too noisy, but I would like to have the highlight for hover range.